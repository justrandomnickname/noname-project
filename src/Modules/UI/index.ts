import { Button } from './components/Button'
import { Title, BigTitle } from './components/Title'
import { RegularText } from './components/RegularText'

export default { Button, Title, RegularText, BigTitle }
