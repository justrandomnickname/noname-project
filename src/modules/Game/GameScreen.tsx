import React from 'react'
import { Wrapper } from '@Game/views/Wrapper'
// import UI from '@Core/UI'
import { Props } from './interfaces/GameScreen'

const GameScreen: React.FC<Props> = (props: Props) => {
  return <Wrapper>{props.children}</Wrapper>
}

export default GameScreen
