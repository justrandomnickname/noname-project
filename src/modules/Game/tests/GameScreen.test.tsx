import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import GameScreen from '@Game/GameScreen'
import UI from '@Core/UI'

it('(GameScreen) renders correctly', () => {
  const wrapper = renderer
    .create(
      <GameScreen>
        <UI.Button />
      </GameScreen>,
    )
    .toJSON()
  expect(wrapper).toMatchSnapshot()
})
