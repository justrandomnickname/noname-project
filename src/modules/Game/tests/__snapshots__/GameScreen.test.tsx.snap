// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(GameScreen) renders correctly 1`] = `
.c0 {
  width: 100%;
  height: 100%;
}

.c1 {
  width: 200px;
  height: 200px;
  border: 2px solid black;
}

<div
  className="c0"
>
  <div
    className="c1"
  />
</div>
`;
